packages/components/src/components/Box/Box.tsx

```diff
@@ -10,9 +11,9 @@ export interface BoxProps extends HTMLAttributes<HTMLDivElement> {
 const Wrapper = styled.div<{ state: BoxProps['state'] }>`
     display: flex;
     flex: 1;
-    border-radius: 8px;
-    padding: 16px 14px;
-    border: solid 1px ${({ theme }) => theme.STROKE_GREY};
+    border-radius: ${borders.radii.sm};
```

Comment:
We don't have `8px` border in design tokens and we have this radius in multiple places. What should we do about it?

Reply:
use `radii.sm` – 12px :) The UI got rounde.